src/packer.pkr.hcl

```
@@ -103,7 +103,13 @@ source "amazon-ebs" "example" {
 }
 
 build {
-  sources = ["source.amazon-ebs.example"]
+  sources = ["source.amazon-ebs.ubuntu_server"]
+
+  # We are adding this to avoid the failure to install aptitude during build time 
+  # A bug for this has been created here: #12
```

Sorry to nitpick, but we prefer to use standard GitHub terminology (issue vs. bug):
```suggestion
  # Issue #12 has been created for this.
```